deepspeed/runtime/dataloader.py

```diff
@@ -41,7 +41,8 @@ def __init__(self,
                  num_local_io_workers=None,
                  data_sampler=None,
                  data_parallel_world_size=None,
-                 data_parallel_rank=None):
+                 data_parallel_rank=None,
+                 drop_last=False):
```

Comment:
Thanks for fixing this issue. But I have a comment. 

If this is hardcoded to False, does it not retain the original problem in #326? I think it would be better to make this a ds_config option that could be passed into DeepSpeedDataLoader [here](https://github.com/microsoft/DeepSpeed/blob/aa1212910c8a067e4014e157627fa80b696e824b/deepspeed/runtime/engine.py#L1153)?

Reply:
I left it default to "False" because in the torch.utils.data, it's "False" by default. Again if anyone encounters an error due to  "dataset size is not divisible by the batch size", and needs to use the feature they can always pass drop_last=True in [here](https://github.com/microsoft/DeepSpeed/blob/aa1212910c8a067e4014e157627fa80b696e824b/deepspeed/runtime/engine.py#L1153).

Is this okay or should I implement it as a ds_config?